habitat-lab/habitat/config/benchmark/rearrange/rearrange_easy_human_and_fetch.yaml

```diff
@@ -3,7 +3,7 @@
 defaults:
   - /habitat: habitat_config_base
   - /habitat/simulator/agents@habitat.simulator.agents.agent_0: depth_head_agent
-  - /habitat/simulator/agents@habitat.simulator.agents.agent_1: depth_head_agent
+  - /habitat/simulator/agents@habitat.simulator.agents.agent_1: rgbd_head_agent
```

Comment:
Why switch to RGBD here? I thought @ASzot was training depth-only Fetch policies.

Reply:
My bad, didn't noticed. I shouldn't be there.